code-challenges.test.js

```diff
@@ -49,9 +107,35 @@ const randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
 const randomNouns2 = ["temperature", "database", "chopsticks", "mango", "deduction"]
 // Expected output: ["Temperature", "Database", "Chopsticks", "Mango", "Deduction"]
 
-
-// b) Create the function that makes the test pass.
-
+describe("upperCase", () => {
+    it("takes in an array of words and returns an array with all the words capitalized.", () => {
+      const randomNouns1 = []
+      const randomNouns2 = []
+      expect(upperCase(randomNouns1))
+      expect(upperCase(randomNouns2))
+    })
+  })
+
+  // Pseudo Code: 
+  // create a function called "upperCase"
+  // place a param called "string"
+  // return the param and use the map () method to access the values
+  // return the value's first character and use the toUpperCase() method
+
+// // b) Create the function that makes the test pass.
+const upperCase = (string) => {
+  return string.map(value=> {
+    return value.charAt(0).toUpperCase() + value.substring(1);
+  })
+}
```

Comment:
Excellent!  I would usually recommend to have your param be named according to the data type that is being passed into your function, in this case an array is the input.